code/obj/item/stun_baton.dm

```diff
@@ -522,3 +522,11 @@ TYPEINFO(/obj/item/baton/ntso)
 		src.is_active = FALSE
 		usr?.show_text("The [src.name] is now open and unpowered.", "blue")
 		src.process_charges(-INFINITY)
+
+/obj/item/baton/ntso/active //active subtypes so NTSC baton spawns ready to go
```

Comment:
May as well just make this the default for the type? Regular batons start on iirc.